docs/research/ed_research_links.md

```diff
@@ -0,0 +1,82 @@
+# Research Links
+
+Work on gathering some research related to the topics discusses in the last client meeting. Will discuss our findings during the next team meeting (Oct 3rd 2022).
+
+## LAZYKH
+
+Youtube animator who designed lip synched animations.
+
+### Rundown
+
+- Animations are representated as a series of images
+
+- Coordinates csv that indicated where the mouth movements should be placed on the animation
+
+- Has pose rules to change the animation image as the text is being read
+  - period indicated a pose change
+
+- Input:
+  - Text file
+    - Each line has an emotion
```

Comment:
does this mean the Microsoft Azure has sentiment analysis built in?

Reply:
No it unfortunately does not :((( 

Looks like will scrap this option since Microsoft requires the usage of their own speech synthesis.